src/containerapp/azext_containerapp/_clients.py

```diff
@@ -359,6 +359,75 @@ def list_usages(cls, cmd, resource_group_name, name):
         r = send_raw_request(cmd.cli_ctx, "GET", request_url)
         return r.json()
 
+    @classmethod
+    def update_httprouteconfig(cls, cmd, resource_group_name, name, httprouteconfig_name, httprouteconfig_envelope):
+        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
+        sub_id = get_subscription_id(cmd.cli_ctx)
+        url_fmt = "{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.App/managedEnvironments/{}/httpRouteConfigs/{}?api-version={}"
+        request_url = url_fmt.format(
+            management_hostname.strip('/'),
+            sub_id,
+            resource_group_name,
+            name,
+            httprouteconfig_name,
+            cls.api_version)
+
+        r = send_raw_request(cmd.cli_ctx, "PUT", request_url, body=json.dumps(httprouteconfig_envelope))
+        return r.json()
+
+    @classmethod
+    def list_httprouteconfigs(cls, cmd, resource_group_name, name, formatter=lambda x: x):
+        route_list = []
+        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
+        sub_id = get_subscription_id(cmd.cli_ctx)
+        url_fmt = "{}/subscriptions/{}/resourceGroups/{}/providers/Microsoft.App/managedEnvironments/{}/httpRouteConfigs?api-version={}"
+        request_url = url_fmt.format(
+            management_hostname.strip('/'),
+            sub_id,
+            resource_group_name,
+            name,
+            cls.api_version)
+
+        r = send_raw_request(cmd.cli_ctx, "GET", request_url, body=None)
+        j = r.json()
+        for route in j["value"]:
```

Comment:
It may throw None Error if  `j["value"]` is `None`.

Reply:
This is the same pattern we use in the other clients. It seems 'value' is always present.

Reply:
I tested 2 cases for it.


```
a= json.loads('{}')
for route in a["value"]:
 print(route)
```
![image](https://github.com/user-attachments/assets/d1a457dd-073b-44c0-a8b9-7ea8f16dc94c)

```
a= json.loads('{"value": null}')
for route in a["value"]:
 print(route)
```
![image](https://github.com/user-attachments/assets/4b49bd35-768a-4270-8533-d6f8739bf841)